styleguide/components/StyleGuide/index.js

```diff
@@ -4,14 +4,16 @@
 // the output components of React Styleguidist's Markdown.
 import { Global, css } from '@emotion/core';
 import PropTypes from 'prop-types';
-import React from 'react';
+import { React, useState } from 'react';
```

Comment:
This should be:

```suggestion
import React, { useState } from 'react';
```

Reply:
I had that in the other file! Weirdly this worked fine, but ok!